src/transformers/models/bert/modeling_tf_bert.py

```diff
@@ -620,13 +745,21 @@ def call(
             position_ids=position_ids,
             head_mask=head_mask,
             inputs_embeds=inputs_embeds,
+            encoder_hidden_states=encoder_hidden_states,
+            encoder_attention_mask=encoder_attention_mask,
+            past_key_values=past_key_values,
+            use_cache=use_cache,
             output_attentions=output_attentions,
             output_hidden_states=output_hidden_states,
             return_dict=return_dict,
             training=training,
             kwargs_call=kwargs,
         )
 
+        # Copied from `modeling_bert.py` (we can't do this in `booleans_processing`)
+        if not self.is_decoder:
+            inputs["use_cache"] = False
+
```

Comment:
It seems we can perform this change inside `booleans_processing`, since config is passed there, and config always has `is_decoder`. Let me know if it would be better to put this block there.